### ISSUE TITLE
The values of "image" and "Network Name" are not displayed when the harvester node template is edited again. 

### ISSUE THREAD

**Issue by TangStone** [harvester/harvester#851]
**Describe the bug**
1. Create a harvester node template
2. Select External Harvester, other configurations are as follows, and create
![image](https://user-images.githubusercontent.com/15829074/117833287-c9833080-b2a8-11eb-98e5-2e3fd0b7053f.png)
3. Go back to the node template page and select the harvester node template you just created to edit
![image](https://user-images.githubusercontent.com/15829074/117833608-0d763580-b2a9-11eb-91af-0ec5e2ef1948.png)
4. Default Internal Harvester, should be External Harvester
![image](https://user-images.githubusercontent.com/15829074/117833973-5f1ec000-b2a9-11eb-8a80-aa8a05799212.png)
5. Switch to "External Harvester", "Image" and "Network Name" are not displayed
![image](https://user-images.githubusercontent.com/15829074/117834797-0a2f7980-b2aa-11eb-99d5-8e140d3d97e7.png)
6. Switch to "Internal Harvester", After switching back to “External Harvester”, "Host","Port","Image" and "Network Name" are not displayed
![image](https://user-images.githubusercontent.com/15829074/117835312-75794b80-b2aa-11eb-8508-15daf3323c1e.png)

**Expected behavior**

Edit the harvester node template to show the values of the parameters that have been created and saved
**Support bundle**

**Environment:**
 - Harvester ISO version: 
 - Underlying Infrastructure (e.g. Baremetal with Dell PowerEdge R630):

**Additional context**

**Comment by WuJun2016** [comment]
@futuretea  @guangbochen  need to build the latest front-end resources to upload to S3

**Comment by guangbochen** [comment]
the CI is updated and the master build can always be getted from https://harvester-node-driver.s3.amazonaws.com/ui/master/component.js

**Comment by futuretea** [comment]
@guangbochen https://github.com/harvester/docker-machine-driver-harvester/pull/17 this PR also need to be merged.

**Comment by guangbochen** [comment]
You should be able to verify this by replacing the Harvester node driver on the [Rancher UI](https://docs.harvesterhci.io/v0.2/rancher/node-driver/#how-to-add-harvester-node-driver-in-dev-mode),
the Harvester node driver and UI assets are available as below:
- https://harvester-node-driver.s3.amazonaws.com/ui/master/component.js
- https://harvester-node-driver.s3.amazonaws.com/driver/master/docker-machine-driver-harvester-amd64.tar.gz

**Comment by juadk** [comment]
Tested successfully in build `master-5ea5fc31-head` ​with ISO mod and 5 nodes cluster.

I was able to reproduce the reported issue.
Then, I updated the harvester node driver with https://harvester-node-driver.s3.amazonaws.com/driver/master/docker-machine-driver-harvester-amd64.tar.gz and https://harvester-node-driver.s3.amazonaws.com/ui/master/component.js
A new node template was created and when I edited it, all the information were still in place as expected:
![Screenshot from 2021-06-08 16-13-42](https://user-images.githubusercontent.com/6025636/121202118-5522b980-c875-11eb-8c23-baee7ac8d5e0.png)
The issue can be closed.
